e2e/specs/interactive-builder.spec.ts

```diff
@@ -224,6 +224,123 @@ test('Create a form using the interactive builder', async ({ page, context }) =>
     const editFormPageURL = page.url();
     formUuid = editFormPageURL.split('/').slice(-1)[0];
   });
+
+  await test.step('Then I click the `Edit` button on the form name I need to edit', async () => {
+    await formBuilderPage.editFormButton().click();
+  });
+
+  await test.step('And then I fill in the updated form name', async () => {
+    await formBuilderPage.editFormNameInput().click();
+    await formBuilderPage.editFormNameInput().fill('An edited form');
```

Comment:
After each update step (either updating/deleting form name, section or question) we should check whether the JSON schema got updated too.